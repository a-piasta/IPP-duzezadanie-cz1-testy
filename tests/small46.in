extendRoute;50;zezpnfxlwnamzsx
getRouteDescription;837
extendRoute;50;obyiiovjznkdfjzhfsumzxffnxl
getRouteDescription;136
getRouteDescription;982
addRoad;juxjmmqjymyazztyxajsfonylqb;obyiiovjznkdfjzhfsumzxffnxl;6;13
getRouteDescription;147
newRoute;231;dklyptkwenidjefqqgfi;hivdftrvddrav
extendRoute;231;sztcttlhfbrdprgqrz
removeRoad;obyiiovjznkdfjzhfsumzxffnxl;juxjmmqjymyazztyxajsfonylqb
getRouteDescription;231
newRoute;799;bkywwahkbnpoihqaudtis;pppqmslumjxwwwnqikuxsz
repairRoad;pppqmslumjxwwwnqikuxsz;juxjmmqjymyazztyxajsfonylqb;13
addRoad;obyiiovjznkdfjzhfsumzxffnxl;zezpnfxlwnamzsx;7;-3
repairRoad;hivdftrvddrav;sztcttlhfbrdprgqrz;-13
repairRoad;dklyptkwenidjefqqgfi;juxjmmqjymyazztyxajsfonylqb;12
extendRoute;799;hivdftrvddrav
repairRoad;obyiiovjznkdfjzhfsumzxffnxl;zezpnfxlwnamzsx;-5
removeRoad;obyiiovjznkdfjzhfsumzxffnxl;juxjmmqjymyazztyxajsfonylqb
extendRoute;231;hivdftrvddrav
addRoad;qmugnuqxingjatnheoplg;obyiiovjznkdfjzhfsumzxffnxl;10;-9
repairRoad;qmugnuqxingjatnheoplg;obyiiovjznkdfjzhfsumzxffnxl;-15
newRoute;20;sztcttlhfbrdprgqrz;obyiiovjznkdfjzhfsumzxffnxl
repairRoad;juxjmmqjymyazztyxajsfonylqb;sztcttlhfbrdprgqrz;-3
getRouteDescription;497
removeRoad;juxjmmqjymyazztyxajsfonylqb;obyiiovjznkdfjzhfsumzxffnxl
getRouteDescription;550
removeRoad;obyiiovjznkdfjzhfsumzxffnxl;juxjmmqjymyazztyxajsfonylqb
extendRoute;20;zezpnfxlwnamzsx
addRoad;sztcttlhfbrdprgqrz;juxjmmqjymyazztyxajsfonylqb;7;-13
addRoad;hivdftrvddrav;pppqmslumjxwwwnqikuxsz;24;-6
repairRoad;pppqmslumjxwwwnqikuxsz;dklyptkwenidjefqqgfi;15
newRoute;959;juxjmmqjymyazztyxajsfonylqb;dklyptkwenidjefqqgfi
removeRoad;zezpnfxlwnamzsx;obyiiovjznkdfjzhfsumzxffnxl
repairRoad;hivdftrvddrav;hivdftrvddrav;-12
repairRoad;dklyptkwenidjefqqgfi;sztcttlhfbrdprgqrz;5
getRouteDescription;959
addRoad;mvqaqnmsmpdspltdglf;sztcttlhfbrdprgqrz;17;-14
removeRoad;obyiiovjznkdfjzhfsumzxffnxl;qmugnuqxingjatnheoplg
addRoad;obyiiovjznkdfjzhfsumzxffnxl;obyiiovjznkdfjzhfsumzxffnxl;25;11
repairRoad;bkywwahkbnpoihqaudtis;hivdftrvddrav;-6
getRouteDescription;81
repairRoad;juxjmmqjymyazztyxajsfonylqb;pppqmslumjxwwwnqikuxsz;-6
extendRoute;231;hivdftrvddrav
newRoute;393;sztcttlhfbrdprgqrz;pppqmslumjxwwwnqikuxsz
removeRoad;juxjmmqjymyazztyxajsfonylqb;obyiiovjznkdfjzhfsumzxffnxl
extendRoute;20;qmugnuqxingjatnheoplg
extendRoute;50;zezpnfxlwnamzsx
removeRoad;obyiiovjznkdfjzhfsumzxffnxl;qmugnuqxingjatnheoplg
getRouteDescription;231
extendRoute;231;pppqmslumjxwwwnqikuxsz
getRouteDescription;393
newRoute;701;dklyptkwenidjefqqgfi;obyiiovjznkdfjzhfsumzxffnxl
extendRoute;393;sztcttlhfbrdprgqrz
extendRoute;393;juxjmmqjymyazztyxajsfonylqb
getRouteDescription;393
newRoute;455;juxjmmqjymyazztyxajsfonylqb;pppqmslumjxwwwnqikuxsz
extendRoute;701;juxjmmqjymyazztyxajsfonylqb
getRouteDescription;231
