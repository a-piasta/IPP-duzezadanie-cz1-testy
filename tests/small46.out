1: NIE
2: 
3: NIE
4: 
5: 
6: TAK
7: 
8: NIE
9: NIE
10: TAK
11: 
12: NIE
13: NIE
14: TAK
15: NIE
16: NIE
17: NIE
18: NIE
19: NIE
20: NIE
21: TAK
22: NIE
23: NIE
24: NIE
25: 
26: NIE
27: 
28: NIE
29: NIE
30: TAK
31: TAK
32: NIE
33: NIE
34: TAK
35: NIE
36: NIE
37: 
38: TAK
39: TAK
40: NIE
41: NIE
42: 
43: NIE
44: NIE
45: NIE
46: NIE
47: NIE
48: NIE
49: NIE
50: 
51: NIE
52: 
53: NIE
54: NIE
55: NIE
56: 
57: NIE
58: NIE
59: 
